compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -282,10 +317,15 @@ private val GenericFontFamiliesMapping: Map<String, List<String>> by lazy {
             mapOf(
                 FontFamily.SansSerif.name to listOf("Noto Sans", "DejaVu Sans", "Arial"),
                 FontFamily.Serif.name to listOf("Noto Serif", "DejaVu Serif", "Times New Roman"),
-                FontFamily.Monospace.name to listOf("Noto Sans Mono", "DejaVu Sans Mono", "Consolas"),
+                FontFamily.Monospace.name to listOf(
+                    "Noto Sans Mono",
+                    "DejaVu Sans Mono",
+                    "Consolas"
+                ),
```

Comment:
Here and in other places in this file: please avoid unrelated changes.
They make the PR longer and harder to review and are potential source of future merge conflicts.